Urania/Urania.Desktop/MainWindow.xaml.cs

```diff
@@ -15,5 +15,17 @@ public partial class MainWindow {
             InitializeComponent();
             this.DataContext = this;
         }
+
+        private void CleanButton_Click(object sender, System.Windows.RoutedEventArgs e) {
```

Comment:
Metoda clean powinna czyścić modele danych, a nie powiązane elementy UI. np.

`WireParameters.Wd = null;`

Reply:
Zamiast is Checked powinno być zmieniane state na domyślne np. WdState. A przy zmianie same comboboxy powinny się zerować.